Assets/Scripts/Map/MapGameState.cs

```diff
@@ -375,8 +377,8 @@ public void Spawn_Bird_Button()
             var forward = new Vector3(cameraForward.x, 0f, cameraForward.z).normalized;
             var rotation = Quaternion.LookRotation(forward);
 
-            // Vector2 playerLocation = new Vector2(21.31624f, -157.858102f);
-            Vector2 playerLocation = new Vector2(Input.location.lastData.latitude, Input.location.lastData.longitude);
+            Vector2 playerLocation = new Vector2(21.31624f, -157.858102f);
+            //Vector2 playerLocation = new Vector2(Input.location.lastData.latitude, Input.location.lastData.longitude);
```

Comment:
Can you change this back to use the latitude and longitude? Or add the check for if we're using unity editor versus mobile